addons/it/ticket/extensions.go

```diff
@@ -94,4 +96,40 @@ var extensions = []*cbc.Definition{
 			},
 		},
 	},
+	{
+		Key: ExtKeyLottery,
+		Name: i18n.String{
+			i18n.EN: "AdE Lottery Code",
+			i18n.IT: "Codice Lotteria AdE",
+		},
+		Desc: i18n.String{
+			i18n.EN: here.Doc(`
+				Lottery key is used to identify the lottery number (Codice lotteria). 
+				This lottery code is provided by the customer at the time of purchase. 
+				It is used to identify the customer in the lottery system provided by the Agenzia delle Entrate.
+			`),
+		},
+		Sources: []*cbc.Source{
+			{
+				Title: i18n.String{
+					i18n.EN: "Agenzia delle Entrate",
+					i18n.IT: "Agenzia delle Entrate",
+				},
+				URL: "https://www.agenziaentrate.gov.it/portale/documents/20143/4952835/Specifiche+Tecniche+Lotteria+Istantanea_V1.pdf/211eae00-0e0e-66b9-a077-895eb0d9fc51",
+			},
+		},
+		Pattern: "^[A-Z0-9]{8}$",
+	},
+	{
+		Key: ExtKeyLine,
+		Name: i18n.String{
+			i18n.EN: "AdE Line Code",
```

Comment:
I'd consider "AdE Line Reference Code"... "ref" implies it is used for referencing as opposed to identifying.